test/provider_os_memory_multiple_numa_nodes.cpp

```diff
@@ -42,7 +42,14 @@ std::vector<int> get_available_cpus() {
     CPU_ZERO(mask);
 
     int ret = sched_getaffinity(0, sizeof(cpu_set_t), mask);
-    UT_ASSERTeq(ret, 0);
+
+    if (ret != 0) {
+        available_cpus.emplace_back(-__LINE__);
```

Comment:
why is `__LINE__` used (here and some other places)?

Reply:
The line number is saved for debugging purposes.

Since get_available_cpus() returns vector<int>, but it does not emplace back negative values, a negative value as a vector element indicates an error.

In retrieve_nodemask, a non-zero value as the first element of the returned pair indicates an error. This non-zero value is an approximate line where the error has occurred.

Should I add these explanations as comments?
Should I change __LINE__ to other values, such as 0, -1 etc.? This is just an approximate line number (since the condition is checked a few lines after the possible error)

Reply:
Changed to -1